kernel/timeout.c

```diff
@@ -100,11 +100,12 @@ static int32_t next_timeout(void)
 	return ret;
 }
 
-void z_add_timeout(struct _timeout *to, _timeout_func_t fn,
-		   k_timeout_t timeout)
+k_ticks_t z_add_timeout(struct _timeout *to, _timeout_func_t fn, k_timeout_t timeout)
```

Comment:
As with z_add_thread_timeout(), I think it is worthwhile to note what this return value means (system tick when timeout is expected to expire).